### ISSUE TITLE
Modularity reading as "0"

### ISSUE THREAD

**Issue by cdgoodwin** [gephi/gephi#1438]
After working perfectly fine for several imports, the Modularity function is not working. It still works when I import prior CSVs, but not for my most recent (which is the same as the prior CSV but with more data). I've got 170 nodes and 197 edges in it. There's nothing different in the formatting of my CSV so far as I can tell -- this is driving me nuts.

The problem appears to be similar to what was brought up in these two unanswered posts 21 days ago:

[Modularity results in java.lang.NullPointerException #1411](https://github.com/gephi/gephi/issues/1411)

[Modularity bug #1412](https://github.com/gephi/gephi/issues/1412)

My own recent log:
[messages.txt](https://github.com/gephi/gephi/files/188410/messages.txt)

**Comment by cdgoodwin** [comment]
I found one duplicate edge in my CSV, and deleted it. Once I did that, the modularity *seems* to be okay again.

**Comment by Humans-Huddle** [comment]
Same issue - persists even with removal of duplicate edges - though I'd imagine these should have increased edge weight?

**Comment by duncdrum** [comment]
@cdgoodwin can you try to import csv (remove double nodes) -> export graph file (.gexf) -> close project;
open -> new project -> import (your gexf file  -> calculate modularity (make a backup gexf file just to be sure).

This has consistently solved my problems with pesky csv and modularity.

**Comment by eduramiba** [comment]
Closing as duplicate of #1419

**Comment by naco-siren** [comment]
@duncdrum In version 0.9.1, [File] -> [Import...] opens an empty import wizard.
![image](https://cloud.githubusercontent.com/assets/8222638/14230504/af7f7146-f98d-11e5-9a68-bb2e1609a422.png)

**Comment by duncdrum** [comment]
@naco-siren sorry my bad, the proper command is file -> open -> GEXF files.
![screenshot 2016-04-03 14 21 38](https://cloud.githubusercontent.com/assets/6205362/14232306/74edc662-f9a7-11e5-8904-08c9a2d81ac7.png)

@eduramiba I understand why we programmatically **import** csv, but **open** .gexf files. But from the user or ui perspective it might make sense to only **open** .gephi projects, and **import** csv, gexf, graphml etc.

**Comment by eduramiba** [comment]
About the empty import wizard, it's only there for import plugins that you might install. I guess that's a bit confusing.

**Comment by cdgoodwin** [comment]
>@cdgoodwin can you try to import csv (remove double nodes) -> export graph file (.gexf) -> close project;
open -> new project -> import (your gexf file -> calculate modularity (make a backup gexf file just to be sure). This has consistently solved my problems with pesky csv and modularity.

Thanks! This worked out just fine. I actually was able to do it without the closing and reloading step. Once I saved the project as a .gexf, the modularity functioned as normal.
